PythonToolbox/quantconnect/api.py

```diff
@@ -468,8 +468,9 @@ def download_data(self, symbol, securityType, market, resolution, date, fileName
         # Make sure the link was successfully retrieved
         if link['success']:
             # download and save the data
-            request = requests.get(link['link'])
+            request = requests_get(link['link'], stream=True)
```

Comment:
I love how such a simple change can make a difference. Was any profiling using to show to show the improvement. I agree that the referenced documentation is reason enough to support this change 👍